catalog-info.yaml

```diff
@@ -0,0 +1,43 @@
+
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+# (Required) Acceptable Values: Component, Resource, System
+# Use `Component` unless you know how backstage works and what the other kinds mean.
+kind: Component
+metadata:
+  # (Required) Must be the name of the repo, without the owning organization.
+  name: 'cypress-e2e-tests'
+  description: "e2e tests written in Cypress for different edX applications"
+  links:
+    - url: "https://github.com/openedx/cypress-e2e-tests"
+      title: "openedx/cypress-e2e-tests"
+      # Backstage uses the MaterialUI Icon Set.
+      # https://mui.com/material-ui/material-icons/
+      # The value will be the name of the icon.
+      icon: "Settings"
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: "feanil"
```

Comment:
I'm not sure @feanil will want this responsibility...

Reply:
Since it is optional, I think I would make it 

```
openedx.org/arch-interest-groups: ""
```

but I'd wait for Feanil to chime in on whether he meant to be the default arch interest group by putting himself in the template.

Reply:
I found reference in other repos @sarina  and it can be 

openedx.org/arch-interest-groups: ""
 
 So have updated that and linting too